Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml

```diff
@@ -152,24 +152,24 @@ script:
       description: ip
       isArray: false
       name: ip
-      required: true
+      required: false
```

Comment:
change this back to be `true` 
the file validation was fix here https://github.com/demisto/content/pull/4255